contracts/core/SettingsManager.sol

```diff
@@ -468,6 +481,10 @@ contract SettingsManager is ISettingsManager, Ownable, Constants {
             ) / positionVault.poolAmounts(_token, _isLong);
     }
 
+    function checkBlackList(address _delegate) public view override returns (bool) {
```

Comment:
do we need to take care of political correctness and change to checkBanList?

Reply:
sure